Solutions/Darktrace/Analytic Rules/CreateAlertFromModelBreach.yaml

```diff
@@ -7,74 +7,89 @@ requiredDataConnectors:
   - connectorId: DarktraceRESTConnector
     dataTypes:
       - darktrace_model_alerts_CL
-queryFrequency: 5m
-queryPeriod: 5m # this might need review
-triggerOperator: gt
-triggerThreshold: 0
```

Comment:
That is because the 'kind' field is set to NRT. Which means it doesn't need any of those fields. 

<img width="215" alt="image" src="https://user-images.githubusercontent.com/86425481/214352906-1d6e260f-b5c7-472f-a3d2-44f87dde2e81.png">

Reply:
adding these back in